app/assets/stylesheets/frontend/views/_history-people.scss

```diff
@@ -1,68 +1,37 @@
-.historic-people-index {
-  .clear-person {
-    clear: both;
+.historic-people-index__section-row-header {
+  @include govuk-media-query($from: tablet) {
+    margin-top: govuk-spacing(2);
+    margin-bottom: govuk-spacing(4);
   }
 
-  .featured-profiles {
-    margin-bottom: $gutter;
+  // unique image card styles for "past-foreign-secretaries" & "past-past-chancellor"
+  .gem-c-image-card__list-item-link {
```

Comment:
This doesn't feel great, overriding component styles. Willing to let it pass if we absolutely have to but I'd suggest considering either/or 1) Checking with design that the default isn't good enough, 2) making this style the default for the "unlinked links".

Reply:
We shouldn't be overriding component styles in applications - it breaks component isolation - e.g. if we were to change the name of this class in the component, this override would suddenly stop working and we wouldn't know about it.

If the link styles definitely need to be different in this one situation, please add it as an option to the component. Do they definitely need to be different?

Reply:
Agreed. 🤔  
Let me do the reverse and reduce the font everywhere.